CHANGELOG.md

```diff
@@ -3,6 +3,38 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [9.1.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/sdk@9.0.0...@agoric/sdk@9.1.0) (2021-09-23)
+
+
+### Features
+
+* **deployment:** use latest faucet-helper.sh from testnet ([83f45f6](https://github.com/Agoric/agoric-sdk/commit/83f45f6be8112b9e74f687d9436963051d9e5308))
```

Comment:
ideally "use latest faucet-helper" would be rephrased to say what the new end-user feature is.